### ISSUE TITLE
Add a page about accessibility standards we support

### ISSUE THREAD

**Issue by Xymmer** [SAP/spartacus-docs#703]
Accessibility section should have a page that outlines the standards we support.
Added in 2.0:
- Full Keyboard Support (WCAG 2.0, 2.1.1, Level A, Keyboard; WCAG 2.0, 2.1.2, Level A, No Keyboard Trap)
- Consistent Navigation (WCAG 2.0, 3.2.3, Level AA, Consistent Navigation)
- Tabbing/Reading Order (WCAG 2.0, 2.4.3, Level A, Focus Order)
- Focus Visible (WCAG 2.0, 2.4.7, Level AA, Focus Visible)
- Grouping 

(ask Miguel for the grouping one)
